ecl/hql/hqlfold.cpp

```diff
@@ -2154,6 +2154,82 @@ IHqlExpression * foldConstantOperator(IHqlExpression * expr, unsigned foldOption
             }
             break;
         }
+    case no_regex_findset:
+        {
+            IValue * t0 = expr->queryChild(0)->queryValue();
```

Comment:
The code this was copied from is old, and also not the best example of code in the system.  However it is probably not worth going back and cleaning it up.